### PR TITLE
Added user authorization templates and routes

### PR DESCRIPTION
**Pull request by Saklavi**
- Added login, password reset/change templates
 - Added authentication routes in users app
 - Extended users table with is_active column
 - Added login, logout url paths in settings

### CODE REVIEW

File: backend/apps/users/templates/users/password_reset_done.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+{% block styles %}
+  <link rel="stylesheet" type="text/css" href="{% static 'users/css/form_style.css' %}">
+{% endblock %}
+
+{% block content %}
+
+	<div class="form form__notification">
+		<p>
+			Well done, now in your mailbox should be mail with password-change link.
```
Comment (rimmaciej) [inline backend/apps/users/templates/users/password_reset_done.html:14]:
```suggestion
			Well done, you should get an e-mail with the password change link.
```

File: backend/apps/users/templates/users/password_reset_complete.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+{% block styles %}
+  <link rel="stylesheet" type="text/css" href="{% static 'users/css/form_style.css' %}">
+{% endblock %}
+
+{% block content %}
+
+	<div class="form form__notification">
+		<p>
+			Your password has been changed. Log in, with new credentials.
```
Comment (rimmaciej) [inline backend/apps/users/templates/users/password_reset_complete.html:14]:
```suggestion
			Your password has been changed successfully! You can log in with new credentials.
```

File: backend/apps/users/templates/users/password_change_done.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+{% block styles %}
+  <link rel="stylesheet" type="text/css" href="{% static 'users/css/form_style.css' %}">
+{% endblock %}
+
+{% block content %}
+
+	<div class="form form__notification">
+		<p>
+			Your password has been successfully changed.
```
Comment (rimmaciej) [inline backend/apps/users/templates/users/password_change_done.html:14]:
```suggestion
			Your password has been changed successfully.
```

**Review by rimmaciej — CHANGES_REQUESTED** [review]
Minor changes

**Review by rimmaciej — APPROVED** [review]

**Review by patbojdol — APPROVED** [review]

**Pull request merged**